x/foundation/README.md

```diff
@@ -489,6 +491,16 @@ The message handling should fail if:
 * the address which receives the coins has no authorization of
   `ReceiveFromTreasuryAuthorization`.
 
+## Msg/GovMint
+
+Massive minting is possible through 'msggovmint' up to 1 time after the chain is started.
+
++++ https://github.com/line/lbm-sdk/blob/66988a235a0e01f7a1ee76d719d585ff35f0d176/proto/lbm/foundation/v1/tx.proto#L221-L225
+
+The message handling should fail if:
+
+* The remaining left count is 0.
```

Comment:
It would be better to add the condition about the authority.
```suggestion

* the authority is not the module's authority.
* The remaining left count is 0.
```